spec/features/add_test_results_spec.rb

```diff
@@ -93,6 +98,22 @@ def fill_in_test_result_submit_form(legislation:, date:, test_result:, file:)
     expect(page).to have_css("h1", text: "Confirm test result details")
   end
 
+  def expect_test_result_form_to_be_blank
+    expect(page).to have_field("Against which legislation?", with: nil)
+    expect(page.find_field("Day").text).to be_blank
+    expect(page.find_field("Month").text).to be_blank
+    expect(page.find_field("Year").text).to be_blank
```

Comment:
Any reason not to use the same capybara method here as above?

```suggestion
    expect(page).to have_field("Day", with: "")
    expect(page).to have_field("Month", with: "")
    expect(page).to have_field("Year", with: "")
```

Reply:
Yes - it doesn’t work 😂